### DISCUSSION TITLE
Using IronPython AEDT shell commands from pyaedt

### DISCUSSION THREAD

**Discussion by ghost** [ansys/pyaedt#3663] [Q&A] [answered]
Are there examples of invoking a AEDT IronPython script within a pyaedt project?

**Comment by ghost** [comment]
I guess my understanding is that what pyaedt itself does, but I'm imagining a subprocess call for things not implemented in pyaedt

**Comment by fwlfHS** [comment, marked as answer]
If there is something not yet implemented in pyaedt, you may access the standard API by using the o-properties in pyaedt, like oeditor, oproject, osolution, ... But be careful mixing those commands.

**Reply by ghost** [reply]
hmm, ok, I'll have to run some tests. Thanks!
